File: Harvard_CS50_Introduction_to_CS/CS50-Pset1-master/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = 0;
    int x = 0;
    while (!(n > 0 && n < 9))
    {
        n = get_int("Height: ");
    }
    for (int i = 1; i < n+1; i++)
    {
       while (x < n-i)
       {
          printf(" ");
          x++;
       }
       while (x < n && x >= n-i)
          {
              printf("#");
              x++;
          }
       printf("\n");
       x = 0;
    }
}

File: CS50_2020/Week6/Mario/More/mario.py
# import get_int from cs50 library
from cs50 import get_int

# Initialize condition
require = False
while not require:
    # prompt the user to the size of the half-pyramid
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        require = True

# go through each line
for i in range(n):
    # print hashes in each line
    print(" " * (n - i - 1) + "#" * (i + 1) + "  " + "#" * (i + 1))